tensorflow_io/core/python/experimental/parse_avro_ops.py

```diff
@@ -130,6 +130,7 @@ def _parse_avro(
     dense_defaults=None,
     dense_shapes=None,
     name=None,
+    avro_num_minibatches=0,
```

Comment:
0 as default for avro_num_minibatches? doesn't it invalidate the check of avro_num_minibatches >= 0?

Reply:
nvm, i thought avro_num_minibatches needs to be positive based on the comment there, didn't read it through.